libpod/runtime_img.go

```diff
@@ -23,9 +23,10 @@ import (
 
 // RemoveContainersForImageCallback returns a callback that can be used in
 // `libimage`.  When forcefully removing images, containers using the image
-// should be removed as well.  The callback allows for more graceful removal as
-// we can use the libpod-internal removal logic.
-func (r *Runtime) RemoveContainersForImageCallback(ctx context.Context) libimage.RemoveContainerFunc {
+// should be removed as well unless the request comes from the docker compat API.
```

Comment:
nit of a nit, only fix if you have other changes, otherwise ignore
```suggestion
// should be removed as well unless the request comes from the Docker compat API.
```

Reply:
Done, thanks Tom